tests/test_forecasts.feature

```diff
@@ -0,0 +1,15 @@
+Feature: /forecasts/
+
+    Scenario: Get forecasts from spotwx
```

Comment:
Need to handle both the scenarios where stations from from wfwx and where it comes from json.